content/articles/lucy-guo/index.md~

```diff
@@ -0,0 +1,108 @@
+---
+title: New Beginnings
+date: "2015-05-28T22:40:32.169Z"
+description: This is a custom description for SEO and Open Graph purposes, rather than the default generated excerpt. Simply add a description field to the frontmatter.
+---
+
+Far far away, behind the word mountains, far from the countries Vokalia and
+Consonantia, there live the blind texts. Separated they live in Bookmarksgrove
+right at the coast of the Semantics, a large language ocean. A small river named
+Duden flows by their place and supplies it with the necessary regelialia.
+
+## On deer horse aboard tritely yikes and much
+
+The Big Oxmox advised her not to do so, because there were thousands of bad
+Commas, wild Question Marks and devious Semikoli, but the Little Blind Text
+didn’t listen. She packed her seven versalia, put her initial into the belt and
```

Comment:
is this file also going to be added? says it's part of the commit but i'm assuming this was just filler from before

Reply:
Hm I thought that was an artifact from my vim editor or something. I'll remove lol. was mainly just testing to see if i could make PRs